### PR TITLE
add check registry command type

### PR DESCRIPTION
**Pull request by refaelm92**
Signed-off-by: refaelm <refaelm@armosec.io>

### CODE REVIEW

**Review by jnathangreeg — DISMISSED** [review]

File: pkg/command/types/v1alpha1/types.go [hunk 1]
```diff
@@ -41,6 +41,7 @@ type OperatorCommandStatus struct {
 	CompletedAt *metav1.Time                `json:"completedAt,omitempty"` // CompletedAt is the time at which the command was completed
 	Executer    string                      `json:"executer,omitempty"`    // Executer is the entity that executed the command
 	Error       *OperatorCommandStatusError `json:"error,omitempty"`       // Error is the error that occurred during the execution of the command (if any)
+	Payload     []byte                      `json:"payload"`               // Payload is the response payload from execution of the command (if any)
```
Comment (amirmalka) [inline pkg/command/types/v1alpha1/types.go]: Please add omitempty

Comment (amirmalka) [reply pkg/command/types/v1alpha1/types.go]: You also need to update the CRD https://github.com/kubescape/helm-charts/tree/main/charts/dependency_chart/operatorcommand-crds

**Review by amirmalka — CHANGES_REQUESTED** [review]
,

File: pkg/command/types/v1alpha1/types.go (hunk 1 above)
Comment (refaelm92) [reply pkg/command/types/v1alpha1/types.go]: https://github.com/kubescape/helm-charts/pull/536

**Review by amirmalka — APPROVED** [review]

**Pull request merged**